dependencies.props

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+1.0.1-prerelease-01401-04<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This happens when the regex doesn't match anything: https://github.com/dotnet/buildtools/blob/41ca51d8256aa68ca4dd64f6d39695a75deb2573/src/Microsoft.DotNet.VersionTools/Dependencies/FileRegexUpdater.cs#L68-L75. I can't tell without digging into it a bit further where this comes from, though--I can do this later today if it isn't critical.